src/posit/connect/resources.py

```diff
@@ -171,29 +176,12 @@ def reload(self) -> Self:
         return self
 
 
-class ActiveFinderMethods(ActiveSequence[T], ABC, Generic[T]):
-    def __init__(self, ctx: Context, parent: Optional[Active] = None, uid: str = "guid"):
-        """Finder methods.
-
-        Provides various finder methods for locating records in any endpoint supporting HTTP GET requests.
-
-        Parameters
-        ----------
-        ctx : Context
-            The context containing the HTTP session used to interact with the API.
-        parent : Optional[Active], optional
-            Optional parent resource for maintaining hierarchical relationships, by default None
-        uid : str, optional
-            The default field name used to uniquely identify records, by default "guid"
-        """
-        super().__init__(ctx, parent)
-        self._uid = uid
-
+class ActiveFinderMethods(ActiveSequence[T], ABC):
```

Comment:
Restoring as class docs

```suggestion
class ActiveFinderMethods(ActiveSequence[T], ABC):
    """Finder methods.

    Provides various finder methods for locating records in any endpoint supporting HTTP GET requests.""""
```

Reply:
Yes, this does stomp the docs added above, but maybe in a week or two we can do a quartodoc overhual session. This will make the boiler plate disappear and documentation could go with its the type definition

Reply:
Sounds great!